ui/components/PromptComponent.js

```diff
@@ -80,6 +84,7 @@ class PromptComponent extends React.Component {
         title : passed.title,
         subtitle : passed.subtitle,
         options : passed.options,
+        showCheckbox : passed.showCheckbox === true,
```

Comment:
You can do this instead
```suggestion
        showCheckbox : !!passed.showCheckbox,
```